_gsocproposals/2018/proposal_DD4hepROOT.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Tessellated geometry support for DD4hep and ROOT/TGeo
+layout: gsoc_proposal
+project:
+- DD4hep
```

Comment:
2 spaces are required before the `-` for these lines to be interpreted as a list.